src/cmds/scripts/pbs_preuninstall

```diff
@@ -61,6 +62,15 @@ server|execution)
         echo "have systemd"
         systemctl disable pbs
         rm -f /usr/lib/systemd/system-preset/95-pbs.preset
+        if command -v python &> /dev/null && [ -f ${pbs_home}/server_priv/hooks/pbs_cgroups.CF ]; then
+            service_name=`cat ${pbs_home}/server_priv/hooks/pbs_cgroups.CF | python -c "import sys, json; print(json.load(sys.stdin)['cgroup_prefix'])"`
```

Comment:
Same here

Reply:
Good point, Vincent. Since we're concerned with removing the service handling cpuset jobs, then it should be on the mom side.